tests/system/PubSub/PubSubTestCase.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * Copyright 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Tests\System\PubSub;
+
+use Google\Cloud\ExponentialBackoff;
+use Google\Cloud\PubSub\PubSubClient;
+
+class PubSubTestCase extends \PHPUnit_Framework_TestCase
+{
+    const TESTING_PREFIX = 'gcloud_testing_';
+
+    protected static $deletionQueue = [];
+    protected static $grpcClient;
+    protected static $restClient;
+    protected static $topic;
+    private static $hasSetUp = false;
+
+    public function clientProvider()
+    {
+        self::setUpBeforeClass();
```

Comment:
Doesn't `setUpBeforeClass` get invoked by the runner? Why do you call it explicitly?